NavigationReactNative/src/TabBar.tsx

```diff
@@ -43,7 +43,7 @@ class TabBar extends React.Component<any, any> {
         return false;
     }
     render() {
-        var {children, barTintColor, selectedTintColor, unselectedTintColor, bottomTabs, scrollable, primary, scrollsToTop} = this.props;
+        var {children, labelVisibility = "auto", barTintColor, selectedTintColor, unselectedTintColor, bottomTabs, scrollable, primary, scrollsToTop} = this.props;
```

Comment:
Please use `defaultProps` to set the default of 'auto'. That way the default value will be set no matter where we access the props from